lib/features/mailbox/data/extensions/list_mailbox_extension.dart

```diff
@@ -12,7 +12,7 @@ extension ListMailboxExtension on List<Mailbox> {
   Map<String, MailboxCache> toMapCache(AccountId accountId, UserName userName) {
     return {
       for (var mailbox in this)
-        TupleKey(mailbox.id!.asString, accountId.asString, userName.value).encodeKey : mailbox.toMailboxCache()
+        TupleKey(accountId.asString, userName.value, mailbox.id!.asString).encodeKey : mailbox.toMailboxCache()
```

Comment:
so we should upgrade db.

Reply:
Yes, we will upgrade hive database to version  10. It implemented.